packages/react-native/ReactAndroid/build.gradle.kts

```diff
@@ -127,14 +127,17 @@ val preparePrefab by
                   )),
               PrefabPreprocessingEntry(
                   "rrc_text",
-                  Pair(
-                      "../ReactCommon/react/renderer/components/text/",
-                      "react/renderer/components/text/")),
+                  listOf(
+                      Pair(
+                          "../ReactCommon/react/renderer/components/text/",
+                          "react/renderer/components/text/"),
+                      Pair(
+                          "../ReactCommon/react/renderer/attributedstring",
+                          "react/renderer/attributedstring"),
+                  )),
               PrefabPreprocessingEntry(
                   "rrc_textinput",
-                  Pair(
-                      "../ReactCommon/react/renderer/components/textinput/",
-                      "react/renderer/components/androidtextinput/")),
+                  Pair("../ReactCommon/react/renderer/components/textinput/androidtextinput", "")),
```

Comment:
This folder doesn't exists and is making the whole build fail.

Reply:
Sorry, the paths got changed between 0.73 and 0.74. _Should_ be fine now.